infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -125,8 +126,10 @@ var ValidOfflineStoreDBStorePersistenceTypes = []string{
 	"redshift",
 	"spark",
 	"postgres",
-	"feast_trino.trino.TrinoOfflineStore",
+	"trino",
```

Comment:
the docs are using "feast_trino.trino.TrinoOfflineStore" in the example configs. Is there an issue tracking the defect? otherwise, you can simply fix it here 😉

Reply:
@dmartinol it appears the docs are out of date ... here's the code where these classes/types are declared -
https://github.com/feast-dev/feast/blob/a36f7e50d97c85595cbaa14165901924efa61cbb/sdk/python/feast/repo_config.py#L87-L100

Reply:
ah ... agree @dmartinol

Reply:
Opened an issue https://github.com/feast-dev/feast/issues/4837